en/docs/choreo-cli/using-personal-access-tokens.md

```diff
@@ -0,0 +1,125 @@
+# Personal Access Tokens Guide for Choreo CLI
+
+## Introduction
+Personal Access Tokens (PATs) allow you to authenticate without using your primary credentials, adding an extra layer 
+of security and ease of management. 
+
+---
+
+## Table of Contents
```

Comment:
We don't need to manually include a toc this way.  Once we have the subsection titles with the proper formatting, the page toc will be generated accordingly